src/main/java/io/srqsoftware/BadgeController.java

```diff
@@ -88,6 +90,29 @@ private boolean validateBadge(Badge badge) {
 		}
 	}
 
+	@RequestMapping(path = "/adminpw", method=RequestMethod.POST)
+	public ResponseEntity<String> addAdminPassword(@RequestBody User user) {
+		if ((user== null) || (user.getPassword() == null) || (user.getPassword().length() < 1)) {
```

Comment:
eventually we should move to validators on domain objects :^) and the inconsistent spacing on the LHS of the user equals check! :1st_place_medal: